lib/remap-functions/src/to_syslog_facility.rs

```diff
@@ -0,0 +1,244 @@
+        }
+
+        auth {
+            args: func_args![value: value!(4)],
+            want: Ok(value!("auth")),
+        }
+
+        syslog {
+            args: func_args![value: value!(5)],
+            want: Ok(value!("syslog")),
+        }
+
+        lpr {
+            args: func_args![value: value!(6)],
+            want: Ok(value!("lpr")),
+        }
+
+        news {
+            args: func_args![value: value!(7)],
+            want: Ok(value!("news")),
+        }
+
+        uucp {
+            args: func_args![value: value!(8)],
+            want: Ok(value!("uucp")),
+        }
+
+        cron {
+            args: func_args![value: value!(9)],
+            want: Ok(value!("cron")),
+        }
+
+        authpriv {
+            args: func_args![value: value!(10)],
+            want: Ok(value!("authpriv")),
+        }
+
+        ftp {
+            args: func_args![value: value!(11)],
+            want: Ok(value!("ftp")),
+        }
+
+        ntp {
+            args: func_args![value: value!(12)],
+            want: Ok(value!("ntp")),
+        }
+
+        security {
+            args: func_args![value: value!(13)],
+            want: Ok(value!("security")),
+        }
+
+        console {
+            args: func_args![value: value!(14)],
+            want: Ok(value!("console")),
+        }
+
+        solaris_cron {
+            args: func_args![value: value!(15)],
+            want: Ok(value!("solaris-cron")),
+        }
+
+        local0 {
+            args: func_args![value: value!(16)],
+            want: Ok(value!("local0")),
+        }
+
+        local1 {
+            args: func_args![value: value!(17)],
+            want: Ok(value!("local1")),
+        }
+
+        local2 {
+            args: func_args![value: value!(18)],
+            want: Ok(value!("local2")),
+        }
+
+        local3 {
+            args: func_args![value: value!(19)],
+            want: Ok(value!("local3")),
+        }
+
+        local4 {
+            args: func_args![value: value!(20)],
+            want: Ok(value!("local4")),
+        }
+
+        local5 {
+            args: func_args![value: value!(21)],
+            want: Ok(value!("local5")),
+        }
+
+        local6 {
+            args: func_args![value: value!(22)],
+            want: Ok(value!("local6")),
+        }
+
+        local7 {
+            args: func_args![value: value!(23)],
+            want: Ok(value!("local7")),
+        }
+
+        invalid_severity_larger_int {
+            args: func_args![value: value!(475)],
+            want: Err("function call error: facility code 475 not valid"),
+        }
+
+        invalid_severity_negative_int {
+            args: func_args![value: value!(-1)],
+            want: Err("function call error: facility code -1 not valid"),
+        }
+
+        invalid_severity_non_int {
+            args: func_args![value: value!("nope")],
+            want: Err(r#"function call error: value error: expected "integer", got "string""#),
```

Comment:
This one was still failing.

```suggestion
            want: Err(r#"value error: expected "integer", got "string""#),
```

Reply:
ahh yeah, cleaned this up as well!